freelist.go

```diff
@@ -171,7 +174,11 @@ func (f *freelist) read(p *page) {
 	copy(f.ids, ids)
 
 	// Make sure they're sorted.
-	sort.Sort(pgids(f.ids))
+	for i, v := range f.ids[:len(f.ids)-1] {
```

Comment:
Thanks for humoring me and writing the bench @funny-falcon! I had ran it through `-gcflags=-m` (where it had mentioned it would inline `sort.Less()`) but realized too late that the it's ability to do that _within_ `sort.IsSorted()` was inhibited by its definition that takes in an interface. No more questions from me here. :)